ui/src/app/shared/conditions/conditions.html

```diff
@@ -35,10 +35,11 @@
                     <tr *ngFor="let c of conditions?.plain; let i = index">
                         <td class="middle-aligned">
                             <div class="field">
-                                <ng-container *ngIf="conditionNames">
+                                <ng-container *ngIf="conditionNames && !c.unknownVariable">
```

Comment:
Is it possible to set a value that is not in the list ? when never run

Reply:
old and deprecated field. I deleted